akka-http-tests/src/test/scala/akka/http/scaladsl/unmarshalling/sse/LineParserSpec.scala

```diff
@@ -15,18 +15,21 @@ final class LineParserSpec extends AsyncWordSpec with Matchers with BaseUnmarsha
 
   "A LineParser" should {
 
-    "parse lines terminated with either CR, LF or CRLF" in {
+    "parse lines terminated with either LF or CRLF" in {
       Source
         .single(ByteString("line1\nline2\rline3\r\nline4\nline5\rline6\r\n\n"))
         .via(new LineParser(1048576))
         .runWith(Sink.seq)
         .map(_ shouldBe Vector("line1", "line2", "line3", "line4", "line5", "line6", ""))
+    }
 
+    "ignore lines terminated with CR" in {
+      // if the line ends with a CR (\r) and there is no next character, this line can't be emitted
```

Comment:
It can ;)

See my previous comment:

> That means that after reading a CR you need to keep track that you've read a CR before and ignore the next character if it is a LF. In our streaming parser you need to keep track of this across chunk boundaries.